mpf/config_spec.yaml

```diff
@@ -1309,9 +1314,40 @@ segment_display_player:
     priority: single|int_or_token|0
     text: single|str|None
     color: list|color|None
-    action: single|enum(add,remove,flash,no_flash,flash_match,set_color)|add
+    action: single|enum(add,remove,flash,no_flash,flash_match,flash_mask,set_color)|add
+    transition: ignore
```

Comment:
Is there some way to use a subconfig here instead of doing it in code? The segment_display_transitions section contains the legal transition config values. My implementation was based on the MC slide transition config validation. It would be nice if the config could be fully evaluated on load instead of at runtime when a transition is created.

Reply:
Currently we do not have that unfortunately. This would be cool. I think this is called tagged union or something.